judge/judgedaemon.main.php

```diff
@@ -607,12 +626,15 @@ function fetch_executable(
     // we have gotten a submission for judging
     $endpoints[$endpointID]["waiting"] = false;
 
+    // refresh config at start of judge run
+    djconfig_refresh();
```

Comment:
nit: I would move this into the judge() function, I think it would be easier to read that way.